build.gradle

```diff
@@ -415,7 +415,7 @@ subprojects {
     }
 
     retry {
-      maxRetries = 4 // 5 attempts in total
```

Comment:
not part of the change?

Reply:
yeah, I missed it, and I will revert it.